GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -79,6 +78,7 @@ private void OnEnable()
             _playInstantSceneTreeTreeView.OnTreeStateChanged += (treeState) =>
             {
                 Config.AssetBundleScenes = treeState;
+                Config.SaveConfiguration(ToolBarSelectedButton.CreateBundle);
```

Comment:
How often is this going to happen? One thing we were thinking about originally was not having to do too much file I/O as modifications happen.

Reply:
This fires anytime the tree data changes. Specifically when a scene is toggled off or on, or when  a row is removed or added.